packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/climatedata/ClimateDataProxy.java

```diff
@@ -12,9 +12,6 @@ public interface ClimateDataProxy {
 	@GetMapping("/status/{uuid}")
 	ResponseEntity<JsonNode> status(@PathVariable("uuid") String uuid);
 
-	@GetMapping("/search/esgf")
```

Comment:
Removing the searching of climate data effectively drops support for climate data as users won't have a way to get this data into terrarium. This is more a problem for @dgauldie to be aware of!

Reply:
Yeah, you know what I'll add this search back in and we can worry about this if we want to clean up climate data. I can see that happening.